### DISCUSSION TITLE
poor MD with the trained model

### DISCUSSION THREAD

**Discussion by cui-hq** [deepmodeling/deepmd-kit#2245] [Q&A]
![d7034a3b9ff71665e8229364af9e149](https://user-images.githubusercontent.com/118978478/212013992-105a58d7-5417-4ad6-b958-8d4d1376a3d7.png)
I want to train a model to perform MD for glucose in water, 300 structures were choosen as training data, 50 structures were choosen as validation data,  there are one glucose and more than 10 water in every structure, the number of water in different structure is different, for example, there are 11 water molecules in the first structure while 14 water molecules in the second structure, the black represents the validation loss while the red represents the training loss, I think the model is sufficient to perform the MD， but when performing the MD, the system goes bad.
![1673511614378](https://user-images.githubusercontent.com/118978478/212015088-537aabe6-e7fb-4f7e-9065-0883891f1fe1.png)
Can someone give me some advise to solve this problem, thank you.

**Comment by njzjz** [comment]
Please read the [checklist in the documentation](https://docs.deepmodeling.com/projects/deepmd/en/v2.2.0.b0/troubleshooting/precision.html) to see if anything can be improved. Thanks!

**Reply by cui-hq** [reply]
Thanks， I use the Gaussian to make the training set, if I use cp2k, can I solve this problem?

**Reply by njzjz** [reply]
I don't see the difference between Gaussian and CP2k. You could first plot a correlation figure like #2252.

**Reply by xuyangmark** [reply]
> Thanks， I use the Gaussian to make the training set, if I use cp2k, can I solve this problem?

I think that CP2k is good at dealing with large systems with many atoms. The diagonalization method is better at calculating conductors. The OT method could deal with small moluculars, but does not support K-points. Your system seems to be very small, which calls for creation of supercell.

**Reply by njzjz** [reply]
I don't think this system is small. A glucose molecule in water should have 50~100 water molecules nearby within the model cutoff. Weirdly, there are only 10 water molecules in the actual structure. I am wondering if there is something wrong when creating structures.
